forms-flow-web/src/components/Form/EditForm/FormEdit.js

```diff
@@ -360,9 +410,69 @@ const Edit = React.memo(() => {
   const backToForm = () => {
     dispatch(push(`${redirectUrl}form/`));
   };
+  const closeHistoryModal = () => {
+    setshowHistoryModal(false);
+  };
+  const fetchFormHistory = (parentFormId, page, limit) => {
+    getFormHistory(parentFormId, page, limit)
+      .then((res) => {
+        dispatch(setFormHistories(res.data));
+      })
+      .catch(() => {
+        setFormHistories([]);
+      });
+  };
+
+  const fetchProcessHistory = (processKey, page, limit) => {
+    getProcessHistory(processKey, page, limit)
+      .then((res) => {
+        dispatch(setProcessHistories(res.data));
+      })
+      .catch(() => {
+        setProcessHistories([]);
+      });
+  };
+  const handleHistory = (type) => {
+    setshowHistoryModal(true);
+    if (type === "WORKFLOW") {
+      setHistory(false);
+      dispatch(setProcessHistories({ processHistory: [], totalCount: 0 }));
+      if (processListData?.processKey) {
+        fetchProcessHistory(processListData?.processKey, pageNo, limit);
+      }
+    } else {
```

Comment:
can we move this to FlowEdit.js as it is related to process @Ajay-aot

Reply:
updated